{{cookiecutter.project_dirname}}/terraform/modules/kubernetes/deployment/main.tf

```diff
@@ -169,10 +169,6 @@ resource "kubernetes_deployment_v1" "main" {
               }
             }
           }
-          env {
```

Comment:
Are we really sure that it is necessary to remove all this block? in the django backend deployment is nedded an environmental variable named `CACHE_URL`, perhaps it should be enough to remove the `var.cache_url` and the `coalesce`, keeping however the definition of the variable `CACHE_URL`. 
But I could also be wrong, from the diff I don't see explicit references to the `CACHE_URL` variable, but I guess that if the intention is to completely remove this block, maybe, something needs to be changed in `settings.py` to point `CACHES` and `CacheURLValue` to ` REDIS_URL`